processing/data_collection/gazette/spiders/ms_campo_grande.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+from dateparser import parse
+import datetime as dt
+
+import scrapy
+
+from gazette.items import Gazette
+
+
+class MsCampoGrandeSpider(scrapy.Spider):
+    MUNICIPALITY_ID = '5002704'
+    name = 'ms_campo_grande'
+    allowed_domains = ['portal.capital.ms.gov.br']
+    start_urls = ['http://portal.capital.ms.gov.br/diogrande/diarioOficial']
+    ms_campo_grande_url = 'http://portal.capital.ms.gov.br/diogrande/diarioOficial'
```

Comment:
Ao invés de definir essa variável, com a mesma URL que está dentro da _start_urls_, você pode definir o método _start_requests_ no lugar da lista _start_urls_ e deixar seu método _parse_ com o conteúdo do _parse_months_.

Dê uma olhada nesse PR para uma explicação de como isso é mais adequado:
https://github.com/okfn-brasil/diario-oficial/pull/40

Reply:
Thanks for the tips @rennerocha. All done.